### PR TITLE
#79 - Simple implementation of NormalizedProjectName

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #79: added `Simple` implementation of `NormalizedProjectName`, used it in `FromFilename`, updated http version.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/pypi-adapter/issues/80#issuecomment-662240037))

@olenagerasimova Pull request [#80](https://github.com/artipie/pypi-adapter/issues/80) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/pypi-adapter/issues/80#issuecomment-662240037))

@olenagerasimova Job `gh:artipie/pypi-adapter#80` is not assigned, can't get performer

File: src/test/java/com/artipie/pypi/NormalizedProjectNameSimpleTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "superProject,superproject",
+        "my-super-project,my-super-project",
+        "One._Two._Three,one-two-three",
+        "agent--007,agent-007"
+    })
+    void normalisesNames(final String name, final String normalized) {
```
Comment (olegmoz) [inline src/test/java/com/artipie/pypi/NormalizedProjectNameSimpleTest.java:54]: @olenagerasimova there is a typo, `normaliSes`

File: pom.xml [hunk 2]
```diff
@@ -71,7 +71,7 @@ SOFTWARE.
     <dependency>
       <groupId>com.artipie</groupId>
       <artifactId>http</artifactId>
-      <version>0.14.2</version>
+      <version>0.15.1</version>
```
Comment (olegmoz) [inline pom.xml:74]: @olenagerasimova do not see how it is related to other changes in this PR, a good change though!

**Comment by olenagerasimova** [comment]
@olegmoz thanks, typo corrected. Http upgrade is not actually reletad to this PR, but I'd keep this change here to avoid one-line-PRs

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#80` is not assigned, can't get performer

**Pull request merged**